devices.js

```diff
@@ -3081,8 +3081,10 @@ const devices = [
         zigbeeModel: ['BoxDIM2 98425031'],
         model: '98425031',
         vendor: 'Nordtronic',
-        description: 'Box Dimmer 2.0',
-        extend: generic.light_onoff_brightness,
```

Comment:
The `extend: generic.light_onoff_brightness` is almost the same as the `supports`, `fromZigbee` and `toZigbee`, was something not working with this configuration?

I'm asking because the `extend: generic.light_onoff_brightness` provides some more features that are currently defined, like `brightness_move` and on/off transitions (https://github.com/Koenkk/zigbee2mqtt/issues/2850)

Reply:
I copied the configuration from the Namron 4512700 and this was working correctly on my iCasa dimmer. I've changed my configuration and tested it with the iCasa dimmer and everything seems to be working fine. Changes have been made.